content/documentation-v1/getting-started.md

```diff
@@ -5,146 +5,134 @@ weight: 6080
+#### 📁 images
+
+This directory is where publication images are stored. The `src` value in the `figures.yaml` file references the files names of the images as they are saved in this directory.
+
+
+#### 📁 styles
+
+This directory includes a `variables.scss` file and a `custom.css` file for directly overriding theme styles (read more in [*Style Customization*](/documentation/styles-customization/)).
 
 ## Files for Developers
 
-Inside each Quire project, you will find the following directories and files. Developers will primarily use the `config.yml` file and the `bin`, `config`, `site`, and `theme` directories.
+Developers will primarily use the `_includes`, `_layouts`, `_plugins` directories and the `config.yaml` file.
 
 ```tx
-📁 bin         <-- Scripts
-📁 config      <-- Secondary/environmental configuration
-📄 config.yml  <-- Main configuration
+📁 _includes
+📁 _layouts
+📁 _plugins
 📁 content
-📁 data
-📄 README.md
-📁 site        <-- The built site output with `quire site`
-📁 static
-📁 themes      <-- Layouts, shortcodes, styles, js
+  📁 _data    <-- Includes config.yaml file
+  📁 _assets  <-- Includes javascript directory
 ```
 
-### 📁 bin
+### 📁 _includes
 
-Currently, it only contains a `deploy.sh` script file for deploying a Quire project to GitHub pages, which you can learn more about in the [*Deploy Your Project*](/documentation/site-deploy/#preview-your-project-with-github-pages).
+### 📁 _layouts
 
-### 📄 config.yml
+### 📁 _plugins
 
-This is a standard, required file for {{< q-def "Hugo" >}} and also for Quire. In Quire, it is used expressly for configuring how Hugo operates, and for defining a number of key values used in Quire {{< q-def "templates" >}}. Users who have worked on other non-Quire/Hugo projects will note that they typically use the `config.yml` file to also store publication metadata. Given the potentially large scope of this metadata in formal digital publications, Quire uses the `publication.yml` file inside the `data` directory instead. Read more in [*Metadata & Configuration*](/documentation/metadata-configuration/).
+### 📁 _data
 
-### 📁 config
+This directory also contains the `config.yaml` file which is used expressly for configuring how Quire operates and for defining a number of key values used in Quire {{< q-def "templates" >}}. Users who have worked on other non-Quire projects will note that they typically use the `config.yaml` file to also store publication metadata. Given the potentially large scope of this metadata in formal digital publications, Quire uses the `publication.yaml` file inside the `_data` directory instead. Read more in [*Metadata & Configuration*](/documentation/metadata-configuration/).
```

Comment:
I might say something like the following, the rest of the paragraph isn't true for 11ty like it was for Hugo. And we should revisit this when the team finishes refactoring the config.yaml file (https://jira.getty.edu/browse/DEV-12846).

> This directory also contains the `config.yaml` file which is used for configuring different layout options built into Quire, and for defining a number of key values used in Quire {{< q-def "templates" >}}.